src/lib/components/Access/Embargoed.js

```diff
@@ -22,33 +29,45 @@ export class Embargoed {
   renderFilesSection() {
     // Same as Restricted
     const filesStyle = {
-      opacity: "0.5",
-      cursor: "default !important"
+      opacity: '0.5',
+      cursor: 'default !important',
     };
-    let filesContent = <p style={{...filesStyle, textAlign: "center"}}><em>The full record is restricted.</em></p>;
+    let filesContent = (
+      <p style={{ ...filesStyle, textAlign: 'center' }}>
+        <em>{i18next.t('The full record is restricted.')}</em>
+      </p>
+    );
     return filesSection(filesStyle, filesContent);
   }
 
   renderMessageSection() {
-    const fmtDate = (
-      this.embargo.date
-      ? DateTime.fromISO(this.embargo.date)
-        .toLocaleString(DateTime.DATE_FULL) // e.g. June 21, 2021
-      : "???"
-    );
+    const fmtDate = this.embargo.date
+      ? DateTime.fromISO(this.embargo.date).toLocaleString(DateTime.DATE_FULL) // e.g. June 21, 2021
+      : '???';
 
-    const text = <>On <b>{fmtDate}</b> the record and the files will automatically be made publicly accessible. Until then, the record and the files can <b>only</b> be accessed by <b>users specified</b> in the permissions.</>;
+    const text = (
+      <>
+        {/* TODO: use of Trans Component for jsx translations */}
```

Comment:
i have added a `TODO` just in case if we are not happy in the future translating `jsx` format like this.

Reply:
Can't you use the string interpolation here? How does it look like the extracted string if you include the html elements?